tests/baselines/reference/computedPropertyNames12_ES5.js

```diff
@@ -22,10 +22,12 @@ var n;
 var a;
 var C = /** @class */ (function () {
     function C() {
-        this[n] = n;
-        this[s + n] = 2;
+        this[_a] = n;
+        this[_b] = 2;
         this["hello bye"] = 0;
     }
-    C["hello " + a + " bye"] = 0;
+    _c = s, _a = n, _d = s + s, _b = s + n, _e = +s, _f = a, _g = true, _h = "hello " + a + " bye";

```

Comment:
That is a lot of temps. We aren't even using some of them because they are uninitialized static properties. We need to preserve the side effects (if they can even have any), but we don't need to generate a temp variable if we don't use it (e.g. `_c`, `_d`, `_e`, and `_g` above. We also don't necessarily need temps for static properties that will only be evaluated *after* we have emitted the class (such as `_h` above).